simple_history/models.py

```diff
@@ -26,9 +27,15 @@
     pre_create_historical_record
 )
 
+import json
+
 registered_models = {}
 
 
+def model_to_dict(model):
```

Comment:
There's an existing `model_to_dict` in `django.forms.models` - seems a bit odd to depend on something from `forms` here but it may be better than reimplementing? The JSON/serialize dance is a bit wonky to me.

Reply:
Primarily it means it's obvious when queries are done, which is useful. It looks like the one in `django.models.forms` also handles many-to-many relationships, which may cause weirdness / unnecessary queries